src/coreclr/src/nativeaot/System.Private.Reflection.Execution/src/Internal/Reflection/Execution/FieldAccessors/RegularStaticFieldAccessor.cs

```diff
@@ -12,12 +12,14 @@ protected RegularStaticFieldAccessor(IntPtr cctorContext, IntPtr staticsBase, in
             : base(cctorContext, fieldTypeHandle)
         {
             StaticsBase = staticsBase;
-            FieldBase = fieldBase;
+            _fieldBase = fieldBase;
```

Comment:
```suggestion
            _fieldFlags = fieldFlags;
```

Might be better because it's no longer just the information about the base of the field.